projects/packages/my-jetpack/_inc/components/welcome-flow/index.tsx

```diff
@@ -43,23 +54,38 @@ const WelcomeFlow: FC = () => {
 			isUserConnected,
 			isSiteConnected,
 		} );
-		setVisible( false );
 		dismissWelcomeBanner();
 	}, [ recordEvent, currentStep, isUserConnected, isSiteConnected, dismissWelcomeBanner ] );
 
 	const handleEvaluation = useCallback(
-		( _values: { [ key in EvaluationAreas ]: boolean } ) => {
+		( values: { [ key in EvaluationAreas ]: boolean } ) => {
 			setIsProcessingEvaluation( true );
-			setTimeout( () => {
-				// TODO: Mock "evaluation": Implement the evaluation endpoint
-				setVisible( false );
-				dismissWelcomeBanner();
-			}, 3_000 );
+			startEvaluation(
```

Comment:
I don't see any tracks related to the submission of this form. I know we are storing the data in the site options but I think we should add tracks at least here (and probably the CTA clicks as well). I'm not sure if a better place for that is here or the PR that introduces the survey, that's up to you

Reply:
Good point, there is no analytics in this code - will add it!